bytecomp/translmod.ml

```diff
@@ -138,21 +143,22 @@ let rec compose_coercions c1 c2 =
   match (c1, c2) with
     (Tcoerce_none, c2) -> c2
   | (c1, Tcoerce_none) -> c1
-  | (Tcoerce_structure (pc1, ids1), Tcoerce_structure (pc2, ids2)) ->
+  | (Tcoerce_structure (pc1, ids1, runtime_fields1), Tcoerce_structure (pc2, ids2, runtime_fields)) ->
       let v2 = Array.of_list pc2 in
```

Comment:
so `runtime_fields` is never used? can we make it more explicit using `_`, or do you need add an assertion ?

Reply:
Yes.